.github/workflows/benchmarks.yml

```diff
@@ -387,3 +410,21 @@ jobs:
           CLICKBENCH_S3_ENDPOINT: ${{ secrets.CLICKBENCH_S3_ENDPOINT}}
           CLICKBENCH_S3_KEY: ${{ secrets.CLICKBENCH_S3_KEY}}
           CLICKBENCH_S3_SECRET: ${{ secrets.CLICKBENCH_S3_SECRET}}
+
+  create-pr:
+    runs-on: ubuntu-latest
+    needs: run-database-bench
+    if: always()
```

Comment:
Nice catch! Let me update that

Reply:
Updated to `always() && github.event.inputs.update_snapshots == 'always'`, since we want the `create-pr` to run regardless of failure in `run-database-bench` and only run when `github.event.inputs.update_snapshots == 'always'`